docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+```
+
+## PUT an object to Your Federation
+Another powerful Pelican client command is the `pelican object put` command. This command does a simple PUT request to add your object to your federation. Since the vast majority of federations will not allow public writes, we will assume for the example that we will need a token to perform these requests. Here is how you can use `pelican object put`:
+
+```bash
+pelican object put <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/file> -t </path/to/token/file>
+```
+> **Note:** you can also specify the federation url here with the `-f` flag, just be sure not to include it in the request URL as the host name if you decide to do so.
+
+## Pelican Object Copy
+As well with Pelican's `object get` and `object put` commands, there is also an older command called `pelican object copy`. This functions the same as the `pelican object get/put` except that it works for both gets and puts. For example, to do an `object get`:
+```bash
+pelican object copy pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+and to do an `object put`:
+
+```bash
+pelican object copy <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/destination> -t </path/to/token/file>
+```
+
+## Utilizing Queries with your URL
+The Pelican client has the ability to select various amount of queries specified in the remote path to an object. The queries the current Pelican client accepts are: ?pack, ?recursive, and ?directread
+
+### ?pack Query
+The Pelican client has the ability to automatically compress/unpack tarballs when uploading and downloading to and from a federation. The user just needs to supply their request URL with a query. For example, to automatically unpack a tarball on downloading, run:
+```bash
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file/file.tar.gz>?pack=tar.gz <local/path/to/file>
+```
+To upload, specify a directory and Pelican will compress it for you:
+```bash
+pelican object put <local/path/to/directory> pelican://<federation-url></namespace-prefix></path/to/directory.tar.gz>?pack=tar.gz
+```
+
+Pelican accepts the following values for the `pack` query:
+- `pack=auto`:
+	- For downloading, auto-detect the file format and unpack (throws error if it is not any detected format).
+	- For uploading, compress using `.tar.xz`.
+- `pack=tar`, `pack=tar.gz`, `pack=tar.xz`, `pack=zip` :
+	- For downloading, throws an error if the specified object is not in the specified format (`tar`, `tar.gz`, `tar.xz`, `zip`, respectively).
+	- For uploading, create the object in the specified format (`tar`, `tar.gz`, `tar.xz`, `zip`, respectively).
+
+### ?recursive Query
+The ?recursive query can be utilized if the desired remote object is a directory. When this query is enabled, Pelican will recursively download all files and subdirectories within the specified directory in the URL. To use this query, run:
```

Comment:
I'm conflicted here, because we should be saying "objects/namespace paths" and not "files/directories", but I don't know that `?recursive` works for anything _other_ than posix backends right now. Maybe we can say something like "When this query is enabled, Pelican will recursively try to download everything beneath the specified path." Of course, that's not entirely true either...

EDIT:
I read a bit further to the point where you describe the behavior of the `-r` flag in the client, and I like the way you've phrased things there when you say "... indicates to Pelican that all sub paths at the level of the provided namespace should be copied recursively."